domains/authorization/Authorization.API/API.UnitTests/API.UnitTests/API.UnitTests.csproj

```diff
@@ -7,17 +7,28 @@
 
         <IsPackable>false</IsPackable>
         <IsTestProject>true</IsTestProject>
+        <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
     </PropertyGroup>
 
     <ItemGroup>
         <PackageReference Include="coverlet.collector" Version="6.0.0"/>
+        <PackageReference Include="FluentAssertions" Version="6.12.0" />
+        <PackageReference Include="Microsoft.EntityFrameworkCore.InMemory" Version="8.0.4" />
```

Comment:
Are we using inmemory db?

Reply:
Removed:

https://github.com/Energinet-DataHub/energy-origin/pull/1059/commits/c88ce25bc88103d8216ab9c66df7a3a33db6cbe6